web3.md

```diff
@@ -1874,6 +1882,24 @@ Mining
          <txOrder> TXLIST </txOrder>
          <txReceipts> TXRECEIPTS </txReceipts>
 
+    rule <k> #initStateTrie => . ... </k>
+         <stateTrie> _ => #stateRoot( <network> NETWORK </network>, SCHED ) </stateTrie>
+         <schedule> SCHED </schedule>
+         <network> NETWORK </network>
+
+    rule <k> #updateStateTrie => #updateStateTrie(OMMERS) ... </k>
+         <ommerBlockHeaders> [ OMMERS ] </ommerBlockHeaders>
+
+    rule <k> #updateStateTrie( [ _ , _ , OMMER , _ , _ , _ , _ , _ , _ , _ ] , REST ) => #updateStateTrie( REST ) ... </k>
+         <touchedAccounts> ... .Set => SetItem(OMMER) ... </touchedAccounts>
+
+    rule <k> #updateStateTrie( .JSONs ) => . ... </k>
+         <schedule> SCHED </schedule>
+         <stateTrie> TREE => #putAccountsInTrie( TREE, Set2List(SetItem(MINER) (ACCTS -Set #precompiledAccounts(SCHED))), <accounts> ACCTSCELL </accounts> ) </stateTrie>
+         <touchedAccounts> ACCTS => .Set </touchedAccounts>
```

Comment:
Why do we need to do `ACCTS -Set #precompiledAccounts(SCHED)` here? Where is `#precompiledAccounts(SCHED)` being added to the `<touchedAccounts>`?

Reply:
If a contract ever calls one of the precompiled contracts it will be added to `<touchedAccounts>`. And we can't do an update on a precompiled contract in the trie because it doesn't have an associated account so it will remove it from the trie otherwise.